programs/oeis/082/A082902.asm

```diff
@@ -1,7 +1,6 @@
 ; A082902:  a(n) = gcd(2^n, sigma(2,n)) = gcd(A000079(n), A001157(n)).
 ; 1,1,2,1,2,2,2,1,1,2,2,2,2,2,4,1,2,1,2,2,4,2,2,2,1,2,4,2,2,4,2,1,4,2,4,1,2,2,4,2,2,4,2,2,2,2,2,2,1,1,4,2,2,4,4,2,4,2,2,4,2,2,2,1,4,4,2,2,4,4,2,1,2,2,2,2,4,4,2,2,1,2,2,4,4,2,4,2,2,2,4,2,4,2,4,2,2,1,2,1,2,4,2,2,8,2,2,4,2,4,4,2,2,4,4,2,2,2,4,4,1,2,4,2,4,2,2,1,4,4,2,4,4,2,8,2,2,4,2,4,4,2,4,1,4,2,2,2,2,2,2,2,2,4,4,4,2,2,4,2,4,1,2,2,8,2,2,4,1,4,2,2,2,4,2,2,4,2,2,2,2,4,4,2,4,4,4,2,8,4,2,2,2,2,8,1,2,2,2,1,4,2,4,4,4,2,2,2,4,8,2,2,4,2,4,4,4,2,4,4,4,4,2,2,1,2,2,4,2,4,8,2,2,2,4,2,4,4,2,4,2,1,2,2,2,4,4,2,4,4
 
-cal $0,54844 ; Number of ways to write n as the sum of any number of consecutive integers (including the trivial one-term sum n = n).
-gcd $0,1073741824
+cal $0,1227 ; Number of odd divisors of n.
+gcd $0,256
```

Comment:
I have the feeling that the correctness of this program relies on the fact that the second argument (256) is a sufficiently large power of 2 (before it was 1073741824, which is also a power of 2). The change is fine, but we need to guide the miner to avoid programs with such large constants.

Reply:
Large constants, I have the same problem in my own genetic algorithm code. In order not to find a local minima with constants, I had to add mechanism to penalize some instructions.

LODA is much cooler than my genetic algorithm thing.